src/components/ErrorMessage.tsx

```diff
@@ -1,10 +1,12 @@
 /** @jsx jsx */
-import { jsx, css } from "@emotion/react";
+import {jsx, css} from "@emotion/react";
 import colors from "../colors";
+import {selectable} from "../commonStyles";
 
 interface ErrorMessageProps {
     show: boolean,
-    message: string
+    message: string,
+    innerElement?: JSX.Element,
```

Comment:
`message` and `innerElement` seem mutually exclusive, so this seems wrong. Not sure what a better design would be.

Reply:
There's always the option to combine them into one ORed field, and then add instance checks in the code. I can do that.

Reply:
9696f526